Jenkinsfile

```
@@ -0,0 +1,71 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Copyright 2019 Joyent, Inc.
+ */
+
+@Library('jenkins-joylib@v1.0.2') _
+
+pipeline {
+
+    agent {
+        label joyCommonLabels(image_ver: '18.4.0')
+    }
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr: '30'))
+        timestamps()
+    }
+
+    parameters {
+        string(
+            name: 'BUILDNAME',
+            defaultValue: '${BRANCH_NAME} master',
+            description:
+                'The hierarchy of agent branch names to include in the shar ' +
+                'archive. By default, we use the branch of ' +
+                'sdc-agents-installer itself, and fallback to "master" ' +
+                'unless this is a "release-*" branch, when we do not fallback.')
```

```suggestion
                'However, if on a "release-*" branch, then the given BUILDNAME is overridden with "$BRANCH_NAME" to avoid a fallback branch for release builds.')
```

... that might be more clear. But what you have is fine too.

---

Yep, that reads better - I'll use your suggestion, thanks.